main.js

```diff
@@ -83,7 +86,47 @@ function addClickTrackingHandlers() {
 	}
 }
 
-export async function init({ parentSelector }) {
+function getCookieByName(name) {
+	const cookies = document.cookie.split(';');
+
+	const cookie = cookies
+		.map((cookie) => cookie.trim())
+		.find((cookie) => cookie.startsWith(name + '='));
+
+	return cookie ? cookie.substring(name.length + 1) : null;
+}
+
+function dispatchAmplitudeExperimentExposureEvent(variant) {
+	const userId = getCookieByName('FTAllocation');
+	const deviceId = getCookieByName('spoor-id');
+
+	if (!userId && !deviceId) {
+		return;
+	}
+
+	return fetch(AMPLITUDE_API_URL, {
+		method: 'POST',
+		body: JSON.stringify({
+			api_key: AMPLITUDE_ANALYTICS_KEY,
+			events: [
+				{
+					event_type: '$exposure',
+					user_id: userId,
+					device_id: deviceId,
+					event_properties: {
+						flag_key: 'browsable-lists',
+						variant
+					}
+				}
+			]
+		})
+	}).catch(() => {
+		// In the future, we can send this error to Sentry
```

Comment:
This was requested by @jamesnicholls in the definitions equivalent PR, so also applies here:

issue: I think we should have some logging (even if it's just on the console) so it can help us debug if the test is not working as expected.

Reply:
Update: all known error conditions will lead to network errors, which will be logged by the browser, so no need to add any additional logging yet. In the future, we should check the response code and log errors to Sentry though.